package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Earn achievements by using Google Chrome",
   "scripts": {
     "test": "jest",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "format": "prettier --write *.js"
```

Comment:
The command `npm run format` seems to format only js files directly under the project root folder.
<img width="408" alt="screen shot 2018-10-26 at 23 52 18" src="https://user-images.githubusercontent.com/9996596/47580859-31d3c600-d97a-11e8-9d2d-c09640eaeb72.png">